README.md

```diff
@@ -42,21 +44,16 @@ dc exec app python manage.py price
 #### Файлы env
 `make deploy-dev` создаст файлы для окружения (env) со стандартными значениями.
 А затем попросит заполнить их.
+
 Пару рекомендаций по заполнению:
 - Генерим случайные: Django secret key, пароли к локальным базам
+- В файлах `shopelectro/docker/env_files/paths` и `shopelectro/docker/.env` определяем путь `REFARM_DIR=path_to_refarm-site`. Интерфейс refarm-site нестабилен, поэтому иногда при разработке фичи сайта
```

Comment:
it would be good to leave `REFARM_DIR=../../refarm-site` as the default value. Some people may want just to follow instruction with minimal customization.


```suggestion
- В файлах `shopelectro/docker/env_files/paths` и `shopelectro/docker/.env`
определяем путь `REFARM_DIR=../../refarm-site` (или другой путь куда вы сделали git clone репозитория refarm-site). Интерфейс refarm-site нестабилен, поэтому иногда при разработке фичи сайта
```

And add newline before "определяем" plz